guide_docs.md

```diff
@@ -0,0 +1,187 @@
+
+
+
+
+# OpenElis Global 2 GSoC'24 guide .
+
+![](https://s3.hedgedoc.org/demo/uploads/cf5e54fa-ec36-48a0-ba54-22fca1649df2.jpeg)
+
+## Introduction
+![](https://)
+
+Getting started Guide for onbarding GSoC'24 prospective contributors in `openElis Global 2`.
+
+We understand how painful it might be for new onboarding contributors to get started to `openElis Global 2` , As the project continues to evolve, it’s essential to ensure that our documentation remains up-to-date and accessible. We understand that navigating a new project can be challenging, especially for prospective contributors. Therefore, this documentation serves as a central hub for all information related to our project, making it easier for newcomers to get started.
+
+## OpenELIS Global 2
+
+This laboratory information management tool rewrite onto Java Spring, and with all new technology and features. Please see our [website](http://www.openelis-global.org/) for more information.
```

Comment:
i would prefer some thing like (This is a ....

Reply:
Oooooh, let me look at this.

Reply:
> i would prefer some thing like (This is a ....) when starting the sentence

@HerbertYiga the type is now resolved and I added other parts.
Thank you!